components/brave_wallet/browser/keyring_service.cc

```diff
@@ -254,7 +247,7 @@ void KeyringService::Bind(
 }
 
 // static
-std::string KeyringService::GetKeyringIdForCoin(mojom::CoinType coin) {
+std::string KeyringService::GetMainKeyringIdForCoin(mojom::CoinType coin) {
   if (coin == mojom::CoinType::FIL) {
     return mojom::kFilecoinKeyringId;
```

Comment:
Can we remove this since we need to have special handling for Filecoin keyring id for mainnet or testnet?
We can change the interface and return `absl::nullopt` if it is Filecoin and this can also prevent other people from calling the wrong function

Reply:
And it is better to name it similar to `GetKeyringIdForCoinNonFIL`